cirrus.conf

```diff
@@ -28,3 +28,9 @@ threshold = 10
 [test-default]
 where = tests/
 
+[build-server]
```

Comment:
given we arent actually building cirrus with jenkins, is this config just for testing?